package.json

```diff
@@ -32,6 +32,7 @@
     "asyncapi-parser": "^0.3.5",
     "commander": "^2.12.2",
     "fs.extra": "^1.3.2",
+    "handlebars": "^4.1.2",
```

Comment:
You should not need Handlebars. We're using Nunjucks now.

Reply:
👍 it was still being used. Removed un-used code.